BuildAndTest.cmd

```diff
@@ -85,7 +83,7 @@ goto ExitFailed
 @REM Run all tests
 SET PASSED=true
 
-mstest /testContainer:bld\bin\Sarif.UnitTests\AnyCPU_%Configuration%\Sarif.UnitTests.dll
+mstest /detail:errormessage /detail:stdout /detail:errorstacktrace /detail:displaytext /detail:traceinfo /detail:outcometext /detail:spoolmessage /testContainer:bld\bin\Sarif.UnitTests\AnyCPU_%Configuration%\Sarif.UnitTests.dll | tee logs.txt
```

Comment:
You might want to revert the additional switches you added. These make sense in the server builds, but they might output too much information when run on the dev machines. Your call though. I'm not really sure what all the new switches do...